uiplib/gui/mainGui.py

```diff
@@ -79,7 +79,8 @@ def next_wallpaper(self):
 
     def prev_wallpaper(self):
         """Preview previous wallpaper."""
-        self.index -= 1
+        self.index = self.index - 1 if self.index != 0 \
```

Comment:
dont use `\` use brackets insetead it looks way more readable lik` (self.index -1 if self.index !=0 
else len(self.images) - 1)`